app/src/main/java/com/konkuk/medicarecall/ui/feature/home/screen/HomeScreen.kt

```diff
@@ -284,176 +299,107 @@ fun HomeScreenLayout(
-                                    .background(main)
-
+                            Spacer(Modifier.height(20.dp))
 
+                            // 초록색 한 줄 요약 카드
+                            Card(
+                                modifier = Modifier.fillMaxWidth(),
+                                shape = RoundedCornerShape(16.dp),
+                                colors = CardDefaults.cardColors(containerColor = cardBackgroundColor)
                             ) {
-                                Row(
-                                    modifier = Modifier.padding(
-                                        horizontal = 20.dp,
-                                        vertical = 40.dp
-                                    ),
-                                    verticalAlignment = Alignment.CenterVertically
+                                Column(
+                                    modifier = Modifier.padding(20.dp)
                                 ) {
-
-
-                                    val balloonText = homeUiState.balloonMessage
-                                    val trimmedText = balloonText.take(45)
-
-
-                                    //말풍선
-                                    Card(
-                                        modifier = Modifier
-                                            //텍스트에 따라 말풍선 늘리기
-                                            .width(196.dp)
-                                            .heightIn(min = 94.dp)
-                                            .zIndex(2f), //겹치는 도형 위로 올림
-                                        colors = CardDefaults.cardColors(containerColor = Color.White),
-                                        shape = RoundedCornerShape(10.dp)
-                                    ) {
-
+                                    Row(verticalAlignment = Alignment.CenterVertically) {
+                                        Image(
+                                            painter = painterResource(id = R.drawable.char_medi), // 예시 아이콘
+                                            contentDescription = "요약 아이콘",
+                                        )
+                                        Spacer(Modifier.width(8.dp))
                                         Text(
-                                            text = trimmedText,
-                                            style = MediCareCallTheme.typography.R_16,
-                                            color = MediCareCallTheme.colors.gray8,
-                                            modifier = Modifier
-                                                .padding(8.dp)
-                                                .background(Color.White)
+                                            text = "한 줄 요약",
+                                            style = MediCareCallTheme.typography.B_20, // Bold 스타일
+                                            color = MediCareCallTheme.colors.g50,
```

Comment:
```suggestion
                                            color = if (hasSummaryData) {
                                              MediCareCallTheme.colors.g50
                                          } else {
                                              MediCareCallTheme.colors.white
                                          }
```
이 부분 분기 처리 필요할 것 같습니다!

Reply:
수정하였습니다!